.github/workflows/gradle-check.yml

```
@@ -31,8 +27,17 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v44
+        with:
+          files_ignore: |
+            release-notes/**
+            .github/**
+            **.md
```

Lets list the specific files that we want to allow, etc...
```suggestion
            README.md
```

---

There are man .md files 
```
ADMINS.md
CHANGELOG-3.0.md
CHANGELOG.md
CODE_OF_CONDUCT.md
CONTRIBUTING.md
DEVELOPER_GUIDE.md
MAINTAINERS.md
README.md
RELEASING.md
SECURITY.md
TESTING.md
TRIAGING.md
```
Do any of these or more runs any kind of validation during gradle check?

---

Makes sense. Modified the list